### DISCUSSION TITLE
DAI 2023-2024 - Practical work 2 - Thomas Vuilleumier and Sebastian Diaz

### DISCUSSION THREAD

**Discussion by MolinaElGringo** [heig-vd-dai-course/heig-vd-dai-course#141] [Show and tell]
We're going to create a hanged man game.

**Comment by Cobora2001** [comment]
Together with me !

**Comment by Cobora2001** [comment]
Here is our release: [Our release](https://github.com/MolinaElGringo/DAI_Pratical_Work2/releases/tag/RenduCorrected)

From this commit: [Last commit](https://github.com/MolinaElGringo/DAI_Pratical_Work2/commit/57c6fc3eae6dc0eeafac73395d8368c6697ba629)
